### PR TITLE
Full response aggregator 

### PR DESCRIPTION
**Pull request by ejhumphrey**
generates an anonymized CSV file of response data, like the following:

sample_key|worker_id|worker_trust|channel|instrument|response
----------|---------|------------|-------|----------|--------
000046_3840|0e3c2c5e|0.8146|80e3|flute|0.0
000046_3840|81f0b272|0.9091|80e3|trumpet|0.0
000046_3840|7278f54b|0.9167|80e3|trumpet|0.0
000046_3840|5f472264|0.7273|548b|trumpet|0.0
000046_3840|d33b7b08|0.7692|80e3|clarinet|0.0
000046_3840|a7542eea|0.8|80e3|clarinet|1.0
000046_3840|bd501044|0.7708|548b|flute|0.0
000046_3840|2c242b46|0.7207|80e3|flute|0.0
000046_3840|1328fba9|0.7692|548b|clarinet|0.0

For reference, this produces a table of:
* 233944 rows
* 2541 unique workers
* 7 unique channels

### CODE REVIEW

**Comment by simondurand** [comment]
@ejhumphrey is this a separate `...labels.csv` file or is this the new `openmic-20k-sparse-labels.csv` file?

**Comment by ejhumphrey** [comment]
I intentionally did not save it, and I'm not sure about seeding the uuid generator (tho it must be, if not time-based) ... I suppose it's fine to keep under lock and key, just in case?

**Comment by ejhumphrey** [comment]
@simondurand this produces the un-aggregated, per-annotator-response labels. The sparse labels file could be derived from the output of this script.

**Comment by julian-urbano** [comment]
@ejhumphrey yeah, I'd keep it under key locally. Always good idea to know where things came from.

File: scripts/parse_full_results.py [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+
+
+def encrypt_field(values, hashlen=8, retries=5):
+    unique_values = set(values)
+    for n in range(retries):
+        hashmap = {val: str(uuid.uuid4()).replace('-', '')[:hashlen]
+                   for val in list(unique_values)}
+
+        if len(hashmap) == len(unique_values):
+            break
```
Comment (bmcfee) [inline scripts/parse_full_results.py]: Why would this condition ever happen?  `len(dict) == len(dict.keys()) == len(set(dict.keys()))`.  Did you mean to check the number of unique hash outputs here?

Comment (ejhumphrey) [reply scripts/parse_full_results.py]: yea, good catch. that was the thinking, as a check on accidentally creating key collisions. Definitely a bug tho, should be `len(set(hashmap.values()))`, will fetch

**Comment by ejhumphrey** [comment]
@bmcfee I fixed the collision logic, and added hash-map writing to disk. lemme know if anything looks off ... I'd like to start adding some kind of tests around all of these scripts, but lemme know whether or not you think that should be blocking on this PR (i mean, it _should_ but, curious to know what you think).

**Comment by ejhumphrey** [comment]
bmcfee sayz it's fine offline

**Comment by ejhumphrey** [comment]
this closes #21

**Pull request merged**